zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -858,47 +858,53 @@ mod test {
         }
         caller_bytecode.append(&bytecode! {
             PUSH32(Address::repeat_byte(0xff).to_word())
-            PUSH2(10000)
+            // gas 10032 for static call
+            PUSH2(10000 + 32*(!is_call as u64))  // 10000 for call
```

Comment:
It would be easier to reason by showing the final value
```suggestion
            PUSH2(if is_call {10000} else {10032})
```